content/2018-munich/talks/panel-discussion-prometheus-long-term-storage-approaches.md

```diff
@@ -4,12 +4,12 @@ title: "Panel Discussion: Prometheus Long-Term Storage Approaches"
 
 ## Panel Discussion: Prometheus Long-Term Storage Approaches
 
-In this panel, we will explore some of the emerging and competing approaches to Prometheus scalable long-term storage.
```

Comment:
This was moved here on purpose as it's the only one where you need context before reading through the list of people & projects.

Reply:
Ah, I thought it was a mistake (and also broke the very crude regex-based parsing :)). I will revert it.